wix-data-v2/wix-data-v2/Items/DataItemsQueryResult.service.json

```diff
@@ -11,6 +11,9 @@
     {
       "name": "currentPage",
       "type": [
+        {
```

Comment:
Same as above - this is hidden in v2 in Velo, and doesn't show in v1 (even though it possibly should appear). Dev team needs to double-check how v1 is maintained.